bin/redditwallpapers

```diff
@@ -124,7 +127,14 @@ def main():
             osxcmd = 'osascript -e \'tell application "System Events" to set picture of every desktop to "' + imagePath + '" \''
             os.system(osxcmd)
         elif platform.system() == 'Windows':
+            #converting the image
```

Comment:
Thank you for your patience. I was in village for past few weeks. 
I found that I wasn't able to setup a png image as wallpaper(Windows 10 64-bit), that's why I converted it.